README.md

```diff
@@ -11,19 +11,25 @@ npx create-solana-dapp@latest
 [![npm version](https://img.shields.io/npm/v/create-solana-dapp?color=yellow)](https://npmjs.com/package/create-solana-dapp)
 [![npm downloads](https://img.shields.io/npm/dm/create-solana-dapp?color=yellow)](https://npmjs.com/package/create-solana-dapp)
 
+This is a CLI that automates the initialization of predefined Solana templates (see below). 
+
 ## Templates
 
 The templates are supported within `create-solana-dapp`:
 
+These templates use [Gill](https://github.com/solana-foundation/gill) / [@solana/kit](https://github.com/anza-xyz/kit), except for those marked :warning: (legacy templates).
+
 - Next.js
-  - [Next.js + Tailwind CSS (no Anchor)](https://github.com/solana-developers/template-next-tailwind)
-  - [Next.js + Tailwind CSS + Anchor Basic Example](https://github.com/solana-developers/template-next-tailwind-basic)
-  - [Next.js + Tailwind CSS + Anchor Counter Example](https://github.com/solana-developers/template-next-tailwind-counter)
+  - [Next.js + Tailwind CSS + Gill (no Anchor)](https://github.com/solana-developers/solana-templates/tree/main/templates/template-next-tailwind)
```

Comment:
Because of the separation of the templates (in the `solana-developers/solana-templates` repo) and the cli (in this repo) I think it makes most sense to link to the template repo, and not the individual templates.

That way, we don't need to manually keep these 2 in sync, which is kind of the goal of the separation.